src/path_manager.c

```diff
@@ -25,6 +27,19 @@
 #include "path_manager.h"
 #include "configuration.h"
 
+/// Directory containing MPTCP sysctl variable entries.
+# define MPTCP_SYSCTL_BASE "/proc/sys/net/mptcp/"
```

Comment:
detail, not sure it is intended but there is a space after the `#`. I don't know how clang formats this but just because I noticed it, I comment it, it doesn't change anything ;-)

Reply:
There is indeed a space there.  It's a personal preference.  I like to keep the leading `#` for preprocessor directives and symbols in the first column, regardless of indentation level, and indent the text following it as appropriate.

Reply:
My mistake.  In this case it shouldn't be there.  I'll fix that.  Thanks!